docs/release.md

```diff
@@ -15,20 +15,34 @@
   - `git push`
   - Create a PR for this change, merge once approved.
 4. Create an internal release tag to mark on the master branch where we forked for the release.
-  - `git tag "${RELEASE}.x"`
-  - `git push origin "${RELEASE}.x"`
+  - `git tag "${COLLECTOR_RELEASE}.x"`
+  - `git push origin "${COLLECTOR_RELEASE}.x"`
 5. Create the release branch with an empty commit and push.
-  - `git checkout -b "release/${RELEASE}.x"`
-  - `git commit --allow-empty -m "Empty commit to diverge ${RELEASE} from master"`
-  - `git push --set-upstream origin "release/${RELEASE}.x"`
+  - `git checkout -b "release-${COLLECTOR_RELEASE}"`
+  - `git commit --allow-empty -m "Empty commit to diverge ${COLLECTOR_RELEASE} from master"`
+  - `git push --set-upstream origin "release-${COLLECTOR_RELEASE}"`
 
 **Tag and create the collector image release**
 
 1. Increment or set the patch number and release environment variables (if not set).
-  - `export PATCH_NUMBER=0`
-  - `export RELEASE=3.8`
-2. Tag and push the release. Check CI to ensure the triggered builds complete.
-  - `git tag "${RELEASE}.${PATCH_NUMBER}"`
-  - `git push origin "${RELEASE}.${PATCH_NUMBER}"`
-3. Create a pull request to update the `COLLECTOR_VERSION` file in the [stackrox/stackrox](https://github.com/stackrox/stackrox/) repo with the newly create release after CI images have been built.
+  - `export COLLECTOR_PATCH_NUMBER=0`
+  - `export COLLECTOR_RELEASE=3.8`
+2. Tag and push the release.
+  - `git tag "${COLLECTOR_RELEASE}.${COLLECTOR_PATCH_NUMBER}"`
+  - `git push origin "${COLLECTOR_RELEASE}.${COLLECTOR_PATCH_NUMBER}"`
+3. Create a corresponding PR in openshift/release. Go to a forked copy of the openshift/release repo
+  - `git checkout master`
+  - `git pull upstream master`
+  - `git checkout -b "release-${COLLECTOR_RELEASE}"`
+  - `cd ci-operator/config/stackrox/collector`
+  - `mv stackrox-collector-master.yaml stackrox-collector-release-${COLLECTOR_RELEASE}.yaml`
```

Comment:
Why not create a new config file for each release instead of copying the master config?

Reply:
I am not clear what you are suggesting here. Looking at the projects in ci-operator/config I see that there are a lot of projects with configs for their main branch and release branches. I am not sure what the reason for this is. I would think it would be cleaner to keep the configs for the releases in the release branches and the config for the master branch in the master branch.

Reply:
I don't think I see a way to have configs for releases in release branches, but perhaps I've misunderstood what you are suggesting. 

As I see it, since we own the contents of the `/ci-operator/.../stackrox/collector/` directory, we can add as many configs as needed without polluting the namespace of branches. Also, the branch name will be tied to the org/branch name used to create the PR, e.g., `JoukoVirtanen:release-3.10`., which includes an association with your account for access to the prow job.

Reply:
I have changed the mv command to cp and added instructions for merging the PR. This will make sure that other people have access to the release configs.